package.json

```diff
@@ -23,14 +23,16 @@
     "lit-element": "^2.5.1"
   },
   "peerDependencies": {
-    "@alaskaairux/design-tokens": "^3.0.0",
-    "@alaskaairux/webcorestylesheets": "^3.0.0",
+    "@alaskaairux/design-tokens": "^3.3.0",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.2",
     "@webcomponents/webcomponentsjs": "^2.4.4",
     "focus-visible": "^5.1.0"
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.3.0",
```

Comment:
```suggestion
```

Reply:
I can delete this because this have dependencies with webcorestyles using at demo/styles.scss in line 125
&.focus-visible { @include auro_focus-hyperlink(sass); }
using https://github.com/import './node_modules/@alaskaairux/webcorestylesheets/dist/essentials'; and https://github.com/import './node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables'; when I change into the new version, it's have error when build.